Tests/SotoCoreTests/AWSClientTests.swift

```diff
@@ -52,42 +52,43 @@ class AWSClientTests: XCTestCase {
         let awsServer = AWSTestServer(serviceProtocol: .json)
         let elg = MultiThreadedEventLoopGroup(numberOfThreads: 1)
         let httpClient = HTTPClient(eventLoopGroupProvider: .shared(elg))
-        defer {
+        try await withTeardown {
+            let config = createServiceConfig(
+                serviceProtocol: .json(version: "1.1"),
+                endpoint: awsServer.address
+            )
+            let client = createAWSClient(
+                credentialProvider: .static(accessKeyId: "foo", secretAccessKey: "bar"),
+                httpClient: httpClient
+            )
+            defer {
+                XCTAssertNoThrow(try client.syncShutdown())
+            }
```

Comment:
Would it make sense to move this out so put the client inside the teardown with the others?

Reply:
done